docker-compose.yml

```diff
@@ -18,11 +18,4 @@ services:
     volumes:
       - db:/var/lib/postgres
     ports: [ "127.0.0.1:55432:5432" ]
-  mysql:
-    image: percona:5.7.22
-    environment:
-      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
-    volumes:
-      - db:/var/lib/mysql
-    ports: [ "127.0.0.1:33306:3306" ]
```

Comment:
I guess removing this was an oversight?

Reply:
<img width="463" alt="image" src="https://user-images.githubusercontent.com/1930175/231764569-6d7b2eb6-9462-4524-9691-9ac008e00a7e.png">

Hi @jorgemanrubia, the `mysql` key is duplicated.